### ISSUE TITLE
deleting a character in the address bar triggers autocomplete

### ISSUE THREAD

**Issue by sheikh-azharuddin** [mozilla-mobile/fenix#17194]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
Open any website which is bookmarked
Click on address bar, delete a character, it will auto complete then press delete twice to delete the 2nd character (first delete to delete the auto complete suggestion character(s) , second delete to delete the actual character) 

So in a nutshell you have to press delete button twice from keyboard to delete a single character

### Expected behavior
While deleting the character from address bar in edit mode it should not auto complete 

### Actual behavior
When deleting a character browser trying to auto complete..thus need to press delete button twice to delete a single character 

### Device information

* Android device: ? Rog3 
* Fenix version: ? Latest nightly 

 
<img src = https://user-images.githubusercontent.com/58127841/102952877-23e84400-44f6-11eb-984f-cf49946c120b.jpg width = 40%>

**Comment by mcarare** [comment]
@sheikh-azharuddin Thank you for reporting this. Did you try using Gboard? Does the same issue occur?

**Comment by hakkikaancaliskan** [comment]
@mcarare same problem with gboard

https://user-images.githubusercontent.com/17825767/102979061-9537ef00-4516-11eb-8d82-ab56a4e372ec.mp4

**Comment by sheikh-azharuddin** [comment]
> @sheikh-azharuddin Thank you for reporting this. Did you try using Gboard? Does the same issue occur?

Yes same issue with gboard
![Uploading Screenshot_20201223-143156304.jpg…]()

**Comment by kbrosnan** [comment]
Please use neutral language when filling out the subject. You have filed a few bugs with `critical` fix needed. We only use critical (P1/S1) for issues that prevent the use of the browser for most users. Crashing on startup is a good example.

**Comment by lobontiumira** [comment]
I was not able to reproduce this issue using a Google Pixel (Android 10), or a Sony Xperia Z5 Premium (Android 7.1.1) on 6/24 Nightly 91.0a1 build.
@sheikh-azharuddin could you retest with your device?
Thank you!

**Comment by sheikh-azharuddin** [comment]
Yes fixed..hence closing the case
